tasks/closePosition/index.ts

```diff
@@ -78,6 +79,8 @@ task('closePosition', 'Close stETH position on AAVE')
       ETH: ADDRESSES.main.ETH,
       WETH: ADDRESSES.main.WETH,
       stETH: ADDRESSES.main.stETH,
+      wBTC: ADDRESSES.main.WBTC,
```

Comment:
I guess you can get the `mainnetAddress` from the `test/addresses`

Reply:
Updated